corehq/apps/api/odata/utils.py

```diff
@@ -64,3 +67,26 @@ def format_odata_property_for_power_bi(odata_property):
 def get_case_odata_fields_from_config(case_export_config):
     export_columns = case_export_config.tables[0].columns
     return [column.label for column in export_columns if column.selected]
+
+
+def record_feed_access_in_datadog(request, config_id, duration, response):
+    config = ExportInstance.get(config_id)
+    username = request.couch_user.username
+    json_response = json.loads(response.content.decode('utf-8'))
+    rows = json_response['value']
+    row_count = len(rows)
+    try:
+        column_count = len(rows[0])
+    except IndexError:
+        column_count = 0
+    datadog_counter('commcare.odata_feed.test_v3', tags=[
```

Comment:
Curious, does `v3` signify something?

Reply:
I've been iterating on this metric on staging and suspect it might go through some more iterations until it's finalized.  At that point I'll remove ```test_*```.